packages/web-components/src/dialog/dialog.styles.ts

```diff
@@ -6,116 +6,98 @@ import {
   colorNeutralBackground1,
   colorNeutralForeground1,
   colorTransparentStroke,
-  fontFamilyBase,
-  fontSizeBase300,
-  fontSizeBase500,
-  fontWeightRegular,
-  fontWeightSemibold,
-  lineHeightBase300,
-  lineHeightBase500,
+  curveAccelerateMid,
+  curveDecelerateMid,
+  curveLinear,
+  durationGentle,
   shadow64,
-  spacingHorizontalS,
-  spacingHorizontalXXL,
-  spacingVerticalS,
-  spacingVerticalXXL,
   strokeWidthThin,
 } from '../theme/design-tokens.js';
 
 /** Dialog styles
  * @public
  */
 export const styles = css`
-  ${display('flex')}
+  @layer base {
+    ${display('flex')}
 
-  :host {
-    --dialog-backdrop: ${colorBackgroundOverlay};
-  }
+    :host {
+      --dialog-backdrop: ${colorBackgroundOverlay};
+      --dialog-starting-scale: 0.85;
+    }
 
-  dialog {
-    background: ${colorNeutralBackground1};
-    border: ${strokeWidthThin} solid ${colorTransparentStroke};
-    z-index: 2;
-    margin: auto auto;
-    max-width: 100%;
-    width: 100vw;
-    border-radius: ${borderRadiusXLarge};
-    box-shadow: ${shadow64};
-    max-height: 100vh;
-    height: fit-content;
-    overflow: unset;
-    position: fixed;
-    inset: 0;
-    padding: 0;
-  }
+    ::backdrop {
+      background: var(--dialog-backdrop, rgba(0, 0, 0, 0.4));
+    }
 
-  dialog::backdrop {
-    background: var(--dialog-backdrop, rgba(0, 0, 0, 0.4));
-  }
+    dialog {
+      background: ${colorNeutralBackground1};
+      border-radius: ${borderRadiusXLarge};
+      border: ${strokeWidthThin} solid ${colorTransparentStroke};
+      box-shadow: ${shadow64};
+      color: ${colorNeutralForeground1};
+      height: fit-content;
+      margin: auto auto;
+      max-height: 100vh;
+      max-width: 100%;
```

Comment:
is max-width necessary if we are hardcoding width to `100vw`

Reply:
It is not necessary! Will remove.